pims/cine.py

```diff
@@ -586,6 +730,97 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def save_image_sequence(self, base_dir=None, im_ext='.tif', fmt='06d',
+            starts_with=1, method='skimage', crop=True, out_range='uint16'):
+        """
+        Burst and save .cine file into image sequence, following selected method.
+
+        Parameters
+        ----------
+        base_dir : None or str, optional.
+            Base directory into which create save subfolder. If None (default), 
+            take the path of the cine file.
+
+        im_ext : str, optional.
+            Single image extension. Default is tif, other values may lead to 
+            unsuspected behaviour (compression, etc).
+
+        fmt : str, optional.
+            String formatting of image labeling.
+
+        starts_with : int, optional.
+            First saved image number, default 1.
+
+        method : str, optional.
+            Choose between 'skimage' (default), 'ffmpeg'.
+            See description of formats in Notes.
+
+        crop : bool, optional.
+            Crop the output image series (requires data to be stored in the .cine).
+
+        out_range : str.
```

Comment:
This option also seems unnecessary; it would be much better for this transformation to be a PIMS pipeline so it can be used elsewhere.